### PR TITLE
Add Mollie to DefaultPaymentGateways

### PR DESCRIPTION
**Pull request by samueljseay**
This adds the Mollie plugin to the set of DefaultPaymentGateways.

There is not a corresponding PR in the Mollie plugin for the auto config just yet.

As part of this I removed the jsx SVG for Mollie and put the svg in assets instead so the front-end/back-end can both reference an image path, since we can't define the rendering of a JSX component from the DefaultPaymentGateways config.

Part of woocommerce/woocommerce#32258 

### Detailed test instructions:

TBD

### CODE REVIEW

File: src/Features/RemotePaymentMethods/DefaultPaymentGateways.php [hunk 1]
```diff
@@ -58,6 +58,30 @@ public static function get_all() {
 					self::get_rules_for_cbd( false ),
 				),
 			),
+			array(
+				'key'        => 'mollie',
```
Comment (joshuatf) [inline src/Features/RemotePaymentMethods/DefaultPaymentGateways.php:62]: I know we're not entirely sure on the path forward here, but maybe we should update this to `mollie_wc_gateway_banktransfer` or one of the other Mollie gateways so the payment gateway REST API doesn't 404.

We could probably even use this gateway to provide the help text needed for this gateway.

File: client/task-list/tasks/payments/methods/index.js [hunk 2]
```diff
@@ -342,7 +341,7 @@ export function getPaymentMethods( {
 		{
 			key: 'mollie',
 			title: __( 'Mollie Payments for WooCommerce', 'woocommerce-admin' ),
-			before: <MollieLogo />,
+			before: <img src={ wcAdminAssetUrl + 'mollie.svg' } alt="" />,
```
Comment (joshuatf) [inline client/task-list/tasks/payments/methods/index.js:344]: Good call updating this 👍

**Review by joshuatf — APPROVED** [review]
Looks good!  Pre-approving pending an optional gateway key change but the WooCommerce Admin side of this is working well.

File: client/task-list/tasks/payments/methods/index.js (hunk 2 above)
Comment (samueljseay) [reply client/task-list/tasks/payments/methods/index.js]: Its weird the other images never had alts either , but it totally should :/ I'll add one

Comment (samueljseay) [reply client/task-list/tasks/payments/methods/index.js]: I think we should be translating the alt too? I'm gonna add a translation call here, but yeah the other images will still be missing them. It seems out of scope to fix that here, but I could open another PR if its desired, seems maybe low value if this is going away.

Comment (samueljseay) [reply client/task-list/tasks/payments/methods/index.js]: Resolved here https://github.com/woocommerce/woocommerce-admin/pull/7092/commits/809c8aa6c26c457f59b2b08887bc2f765376a5d7

File: src/Features/RemotePaymentMethods/DefaultPaymentGateways.php (hunk 1 above)
Comment (samueljseay) [reply src/Features/RemotePaymentMethods/DefaultPaymentGateways.php]: Resolved here https://github.com/woocommerce/woocommerce-admin/pull/7092/commits/6557c4d2ba76c3bea7d40adf8999bb406c1c52f2

**Pull request merged**